config/swagger.yaml

```diff
@@ -218,6 +218,31 @@ paths:
           schema:
             $ref: '#/definitions/Error'
       security: []
+  /name:
+    get:
+      tags:
+        - external
+      operationId: GetName
+      description: 'Get name entry from Naming System'
+      consumes:
```

Comment:
Since [this commit](https://github.com/aeternity/epoch/commit/7d45685f9e1faab407ec01117749c9824406c2eb) of @ThomasArts , we should not use a `consumes` clause in GETs